_migratingLessonsToDocuments.txt

```diff
@@ -0,0 +1,8 @@
+To do list for migrating the Aila package from just handling lessons, to allow it it handle different types of document.
+
+## Snapshots
+
+We need to generate a JSON schema for each document type.
+
+## Schemas for documents
+The document schemas should be in the Aila package, and move out of protocol.
```

Comment:
Should we move this to notion? Maybe it belongs in an epic or an Aila Engineering Knowledge age

Reply:
Ah yes - I spotted this had slipped in. It was a temporary file I thought I'd ignored!